File: test/test02.pl
#!/usr/bin/perl -w
# Test For Statements
$i = 0;
@array = ();
foreach $i (1..9) {
	print $i;
	push @array, $i
}
print "\n";
print "Array Default Print\n";
print @array;

for($i = 0; $i < 9 && $i > -1; $i++) {
	print $i;
}
print "\n";

print "\n";
print "Iterative Array Print\n";
foreach $i (@array) 
{
	print $i;
}

print "\n";

File: test/test02.py
#!/usr/bin/python2.7 -u
# Test For Statements
i = 0
array = []
for i in xrange(1, 10) :
    print i,
    array.append(i)
print
print "Array Default Print"
print array

i = 0
while (i < 9  and  i > -1) :
    print i,
    i += 1

print

print
print "Iterative Array Print"
for i in array :
    print i,

print
